Doc/library/venv.rst

```diff
@@ -426,7 +426,7 @@ subclass which installs setuptools and pip into a created virtual environment::
                                                          'more target '
                                                          'directories.')
             parser.add_argument('dirs', metavar='ENV_DIR', nargs='+',
-                                help='A directory in which to create the
+                                help='A directory in which to create the'
                                      'virtual environment.')
```

Comment:
You need a space before the apostrophe.

Reply:
Oh yes. But the sample code was giving an error so I just sent a change to resolve it. Ill add the space